Scripts/script-CheckWhitelist.yml

```diff
@@ -41,3 +41,5 @@ outputs:
 scripttarget: 0
 dependson: {}
 timeout: 0s
+deprecated: true
+releaseNotes: "deprecated CheckWhitelist script"
```

Comment:
Should start with capital letter and you should mention the new script to use instead